lib/adequate_crypto_address/doge.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module AdequateCryptoAddress
+  class Doge < Cryptos
+
+    def valid_prefix?
+      # must begin with 'D'
+      # must precede with a capital letter or number
+      /\AD+[A-Z0-9]/.match?(address) && valid_length?
+    end
+
+    def valid_length?
+      # must be at least 34 char
+      address.to_s.length <= 34
```

Comment:
This validation isn't correct if an address needs 34 or more chars. 

```suggestion
      address.to_s.length >= 34
```